content/v2/domains/dnssec.markdown

```diff
@@ -0,0 +1,238 @@
+          -X DELETE \
+          https://api.dnsimple.com/v2/1010/domains/example.com/dnssec
+
+### Response
+
+Responds with HTTP 204.
+
+~~~json
+<%= pretty_print_fixture("/disableDnssec/success.http") %>
+~~~
+
+Responds with HTTP 428 if DNSSEC is not currently enabled.
+
+
+## Get DNSSEC {#get}
+
+    GET /:account/domains/:domain/dnssec
+
+Get the status of DNSSEC, indicating whether it is currently enabled or disabled.
+
+### Example
+
+Get the DNSSEC status for the domain `example.com` in the account `1010`:
+
+    curl  -H 'Authorization: Bearer <token>' \
+          -H 'Accept: application/json' \
+          https://api.dnsimple.com/v2/1010/domains/example.com/dnssec
+
+### Response
+
+Responds with HTTP 200.
+
+~~~json
+<%= pretty_print_fixture("/getDnssec/success.http") %>
+~~~
+
+
+## List delegation signer records {#ds-record-list}
+
+    GET /:account/domains/:domain/ds_records
+
+List delegation signer records for the domain in the account.
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:account` | `integer` | The account id
+`:domain` | `string`, `integer` | The domain name or id
+
+### Example
+
+List all delegation signer records for the domain `example.com` in the account `1010`:
+
+    curl  -H 'Authorization: Bearer <token>' \
+          -H 'Accept: application/json' \
+          https://api.dnsimple.com/v2/1010/domains/example.com/ds_records
+
+### Response
+
+Responds with HTTP 200.
+
+~~~json
+<%= pretty_print_fixture("/listDelegationSignerRecords/success.http") %>
+~~~
+
+### Sorting
+
+For general information about sorting, please refer to the [main guide](/v2/#sorting).
+
+Name | Description
+-----|------------
+`id` | Sort delegation signer records by ID
+`created_at` | Sort delegation signer records by creation date
+
+The default sorting policy is by ascending `id`.
+
+
+## Create a delegation signer record {#ds-record-create}
+
+You only need to create a delegation signer record manually if your domain is registered with DNSimple but hosted with another DNS provider that is signing your zone. To enable DNSSEC on a domain that is hosted with DNSimple, use the DNSSEC enable endpoint.
+
+    POST /:account/domains/:domain/ds_records
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:account` | `integer` | The account id
+`:domain` | `string`, `integer` | The domain name or id
+
+### Example
+
+Create a delegation signer record under the domain `example.com` in the account `1010`:
+
+    curl  -H 'Authorization: Bearer <token>' \
+          -H 'Accept: application/json' \
+          -H 'Content-Type: application/json' \
+          -X POST \
+          -d '<json>' \
+          https://api.dnsimple.com/v2/1010/domains/example.com/ds_records
+
+### Input
+
+Name | Type | Description
+-----|------|------------
+`algorithm` | `string` | **Required**.
+`digest` | `string` | **Required**.
+`digest_type` | `string` | **Required**.
+`keytag` | `string` | **Required**.
+
```

Comment:
We should document `algorithm`, and `digest_type` with an explanation of these numbers, like we did here: https://developer.dnsimple.com/v2/tlds/#tld-types